content/pages/datamanagement/demo.rst

```diff
@@ -91,24 +118,19 @@ Conversion to BIDS
 In order to get a BIDS dataset from the raw dataset, create a new dataset and
 set it up to become a BIDS dataset::
 
-  datalad create bids
-  cd bids
-  datalad run-procedure setup_bids_dataset
+  % datalad create bids
+  % cd bids
+  % datalad run-procedure setup_bids_dataset
 
 Now, install input data as a subdataset::
 
-  datalad install --dataset . --source ../study_ds sourcedata
-  datalad install sourcedata/code/toolbox
-
-
-If the specification wasn't altered, the actual conversion is done by::
-
-  datalad hirni-spec2bids sourcedata/02/studyspec.json
+  % datalad install --dataset . --source ../my_raw_dataset sourcedata --recursive
 
-Note, that this command takes a list of specification files (each expected to be
-an acquisition specification) and converts the respective acquisitions. If you
-want to convert a dataset with multiple acquisitions at once, just use::
+The actual conversion is based on the specification files in the study dataset. You can convert a single one of them (meaning: Everything such a file specifies) or an arbitrary number, including everything at once, of course::
 
-  datalad hirni-spec2bids sourcedata/*/studyspec.json
+  % datalad hirni-spec2bids --anonymize sourcedata/**/studyspec.json
```

Comment:
Also: double `*` should not be needed.

Reply:
I end up with

```
% git st
On branch master
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        .datalad/runinfo/
        README
        dataset_description.json

nothing added to commit but untracked files present (use "git add" to track)

```

I do not manually enter metadata via the webUI.

Reply:
The last to comments kind of belong together, I guess. The double star is needed, since we do have a studyspec.json at the toplevel as well. Now, if you didn't include that, dataset_description.json and README weren't (copy-)converted. Instead heudiconv generated them (nothing was there). At least, that's my guess. However, our heudiconv procedure should discover that and kill those.

Uncommitted run sidecar files: Known issue, but I'm not clear about it yet.

Reply:
FTR: toolbox procedure should now delete heudiconv's template files correctly.